.eslintrc.cjs

```diff
@@ -25,7 +24,7 @@ module.exports = {
       },
     },
   },
-  ignorePatterns: ["dist"],
+  ignorePatterns: ["dist", "cypress"],
```

Comment:
is this on purpose?

Reply:
https://github.com/neolution-ch/react-hook-form-components/blob/main/.eslintrc.cjs
i think its needed because cypress has its own lint config.. without this it throws a couple of this errors:
![image](https://github.com/neolution-ch/react-pattern-ui/assets/103021062/3a49b8aa-c6f6-4b9a-aed7-ed5a2151d797)